font/src/directwrite/mod.rs

```diff
@@ -215,3 +356,51 @@ impl ::std::fmt::Display for Error {
         }
     }
 }
+
+fn em_size(size: Size) -> f32 {
+    size.as_f32_pts() * (96.0 / 72.0)
+}
+
+fn encode_utf16(c: char) -> Vec<u16> {
+    let mut buf = vec![0u16; c.len_utf16()];
+    c.encode_utf16(&mut buf);
+    buf
+}
+
+impl From<Weight> for FontWeight {
+    fn from(other: Weight) -> FontWeight {
+        match other {
+            Weight::Bold => FontWeight::Bold,
+            _ => FontWeight::Regular,
+        }
+    }
+}
+
+impl From<FontWeight> for Weight {
+    fn from(other: FontWeight) -> Weight {
+        match other {
+            FontWeight::Bold => Weight::Bold,
+            _ => Weight::Normal,
+        }
+    }
+}
```

Comment:
Do we need both here, or is this just for completeness? Generally I think it's nicer to have one 'target' format that is used everywhere and we convert to that as soon as possible.

If we need to pass one of our `Weight`s into the `dwrote` crate and we get a `FontWeight` out of `dwrote`, which we need to convert back to our weight, this is obviously the way to go though.

Same applies for `Slant`/`FontStyle` too of course.

Reply:
I've gone for just `Weight` -> `dwrote::FontWeight` etc.